config/locales/views/users/procedure_footer/fr.yml

```
@@ -1,25 +1,41 @@
 fr:
   users:
     procedure_footer:
-      aria_label: "En savoir plus sur %{procedure_name}"
       managed_by:
-        header: 'Cette démarche est gérée par :'
+        header: 'Cette démarche est gérée par'
       contact:
-        header: 'Poser une question sur votre dossier :'
+        header: 'Poser une question sur votre dossier'
         in_app_mail:
-          prefix: 'Directement :'
-          link: "par la messagerie"
+          link: "Directement par la messagerie"
         email:
-          prefix: 'Par email :'
+          link: "Directement par courriel : %{service_email}"
         phone:
-          prefix: 'Par téléphone :'
+          link: 'Par téléphone au %{service_telephone}'
         schedule:
-          prefix: 'Horaires : '
+          prefix: "Horaires d'ouverture : "
         stats:
-          prefix: 'Statistiques :'
-          cta: "voir les statistiques de la démarche"
+          link: "Voir les statistiques de la démarche"
       legals:
-        header: "Cadre juridique :"
+        header: "Cadre juridique"
         data_retention: "Dans %{application_name} : %{duree_conservation_dossiers_dans_ds} mois"
```

espace insécable, sinon ça merdouille sur petits écrans

```suggestion
        data_retention: "Dans %{application_name} : %{duree_conservation_dossiers_dans_ds} mois"
```